### PR TITLE
Create Polymorph.yml

### PR DESCRIPTION
**Pull request by acenturyandabit**
A new project for newbies! I'm trying to make it super friendly for new people :)
This is my personal organisation / task management app which runs off pure javascript (so no angular, vue, whatever experience required - but I am thinking of adding a bridge for later, if possible).

I'm an end user developer and I'm hoping that there are others out there who are of the same mindset and would be willing to contribute!

### CODE REVIEW

File: _data/projects/Polymorph.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: Polymorph
+desc: Ultra-flexible self-hostable organisation tool (and so much more)
+site: acenturyandabit.github.io/polymorph
```
Comment (ritwik12) [inline _data/projects/Polymorph.yml]: @acenturyandabit Please add complete URL with http/https otherwise Travis will fail for this PR.

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/Polymorph.yml (hunk 1 above)
Comment (acenturyandabit) [reply _data/projects/Polymorph.yml]: gotcha, done.

File: _data/projects/Polymorph.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+desc: Ultra-flexible self-hostable organisation tool (and so much more)
+site: https://acenturyandabit.github.io/polymorph/
+tags:
+- Javascript
+- UI
+- flexibility
+- framework
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/acenturyandabit/polymorph/issues?q=is%3Aissue+is%3Aopen+label%3Aup-for-grabs
```
Comment (ritwik12) [inline _data/projects/Polymorph.yml]: @acenturyandabit Please change this to https://github.com/acenturyandabit/polymorph/labels/up-for-grabs

**Review by ritwik12 — CHANGES_REQUESTED** [review]

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@acenturyandabit Thanks!!

**Pull request merged**